pyproject.toml

```diff
@@ -160,6 +165,8 @@ exclude = ["docs", "docs/**"]
 [tool.hatch.build.targets.wheel]
 exclude = ["docs", "docs/**"]
 
+[tool.hatch.metadata]
```

Comment:
@RasmusML without this `pip install -e .[hyformer] fails` for some reason

Reply:
It seems to be required if a dependency is defined by an URL.